play-json/jvm/src/main/scala/play/api/libs/json/jackson/JacksonJson.scala

```diff
@@ -108,12 +106,13 @@ private[jackson] case class ReadingList(content: mutable.ArrayBuffer[JsValue]) e
 }
 
 // Context for reading an Object
-private[jackson] case class KeyRead(content: ListBuffer[(String, JsValue)], fieldName: String) extends DeserializerContext {
```

Comment:
You can add MiMa exclusions for these since these are `private[jackson]` and not part of the public api.